tests/dummy/app/templates/elements/article.hbs

```diff
@@ -1,11 +1,3 @@
-<article class='post' id='post_3'>
-This is an article for John Doe.
-</article>
-
-<article class='post' id='post_1'>
-This is an article for Jane Doe.
-</article>
-
-<article class='post' id='post_2'>
-This is another article for Jane Doe.
-</article>
+<article class="post" id="post_3"> This is an article for John Doe.</article>
+<article class="post" id="post_1"> This is an article for Jane Doe.</article>
+<article class="post" id="post_2"> This is another article for Jane Doe.</article>
```

Comment:
Since `article` is a block element, I put the contents on a new line, but I do think the single line version is very readable. What are your thoughts on this @Ctannerweb – should the content of block elements (`div`, `section`, etc.) always be on a new line?